app/src/main/java/com/sdp/movemeet/view/navigation/Navigation.java

```diff
@@ -113,18 +105,19 @@ public static void goToHome(View view) {
      * Starts ActivityList activity
      * @param view the View in which to start the activity
      */
-    public static void goToListOfActivities(View view) {
+    /*public static void goToListOfActivities(View view) {
```

Comment:
It should be deleted rather than commented I think

Reply:
Yes we will do a new pull for clean the code